src/snowflake/snowpark/_internal/analyzer/datatype_mapper.py

```diff
@@ -140,6 +144,9 @@ def to_sql(
     if isinstance(datatype, VectorType):
         if value is None:
             return f"NULL :: VECTOR({datatype.element_type},{datatype.dimension})"
+    if isinstance(datatype, VectorType):
+        if value is None:
+            return "NULL :: FILE"
```

Comment:
Is it confirmed that this will work?

Reply:
I just tried.`select null::file` raises "invalid type [CAST(null AS FILE)] for parameter 'TO_FILE'"

Reply:
to_file(null) seems to work.

Reply:
fixed